application/common/config/local.test.php

```diff
@@ -1,6 +1,43 @@
 <?php
 
 return [
+    'params' => [
+        'password' => [
+            'minLength' => [
+                'value' => 10,
+                'phpRegex' => '',
+                'jsRegex' => '.{10,}',
+                'enabled' => true
+            ],
+            'maxLength' => [
+                'value' => 255,
+                'phpRegex' => '',
+                'jsRegex' => '.{0,255}',
+                'enabled' => true
+            ],
+            'minNum' => [
+                'value' => 2,
+                'phpRegex' => '',
+                'jsRegex' => '(\d.*){2,}',
+                'enabled' => true
+            ],
+            'minUpper' => [
+                'value' => 1,
+                'phpRegex' => '',
+                'jsRegex' => '([A-Z].*){0,0}',
```

Comment:
I think so, but since that is in the test file and for js I think it is just a copy/paste artifact.